src/deploy/functions/release/planner.ts

```diff
@@ -92,37 +99,33 @@ export function calculateUpdate(want: backend.Endpoint, have: backend.Endpoint):
 
 /**
  * Create a plan for deploying all functions in one region.
- * @param want the desired state
```

Comment:
Should we still document what's in `PlanArgs` here?

Reply:
I think going to `PlanArgs` to read its documentation works . WDYT?